components/script/dom/bluetooth.rs

```diff
@@ -145,7 +144,7 @@ fn convert_request_device_options(options: &RequestDeviceOptions, global: Global
     }
 
     let mut optional_services = vec!();
-    if let Some(opt_services) = options.optionalServices {
+    if let Some(opt_services) = options.optionalServices.clone() {
```

Comment:
Here we need the .clone() because optionalServices is a Vec\<DOMString> and the DOMString does not implement the Copy trait only the Clone trait